flink-connector-jdbc-core/src/main/java/org/apache/flink/connector/jdbc/core/database/catalog/AbstractJdbcCatalog.java

```diff
@@ -576,16 +569,6 @@ protected String getSchemaTableName(ObjectPath tablePath) {
         throw new UnsupportedOperationException();
     }
 
-    /**
-     * URL has to be without database, like "jdbc:dialect://localhost:1234/" or
-     * "jdbc:dialect://localhost:1234" rather than "jdbc:dialect://localhost:1234/db".
-     */
-    protected static void validateJdbcUrl(String url) {
-        String[] parts = url.trim().split("\\/+");
-
-        checkArgument(parts.length == 2);
-    }
-
```

Comment:
In the case of Spanner, the URL is as follows, so I have deleted this validation.
```
jdbc:cloudspanner://hostname/projects/gcp_project_id/instances/instance_id/databases/database_id
```